src/main/java/org/venice/beachfront/bfapi/geoserver/BasicAuthHeaders.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2016, RadiantBlue Technologies, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package org.venice.beachfront.bfapi.geoserver;
+
+import org.apache.tomcat.util.codec.binary.Base64;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile({ "basic-geoserver-auth" })
+public class BasicAuthHeaders extends HttpHeaders implements AuthHeaders {
+
+	private static final long serialVersionUID = 1L;
+
+	@Value("${vcap.services.pz-geoserver-efs.credentials.geoserver.username}")
```

Comment:
This needs to be changed to pz-geoserver.

We are no longer using pz-geoserver-efs

Reply:
You can see the VCAP properties for pz-geoserver in the l2 pcf apps manager

Reply:
Done